examples/rust/components/blobby/Cargo.toml

```diff
@@ -14,7 +14,7 @@ name = "blobby"
 anyhow = "1.0.81"
 form_urlencoded = "1.2.1"
 http = "1.1.0"
-wasi = "0.13.1"
+wasi = "=0.13.1"
```

Comment:
A note here and in the other spots would be great so we can remember *why*

Reply:
https://github.com/wasmCloud/wasmCloud/pull/2808/commits/d66a86f9c8131677cac3f0b7bf1bb6afe6b82b78